### PR TITLE
Docs: Update docs for monorepo changes

### PR DESCRIPTION
**Pull request by jackw**
This PR does the following:

- [x] Changes the references in readme and other files from create-plugin to plugin-tools
- [x] Improves on contributing guide to help community understand expectations, workflows and how to contribute
- [x] Adds specific contributing guides for each package
- [x] Introduces PULL_REQUEST and ISSUE_TEMPLATES and adds config so new features are discussions

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+
+<!--
+Please run `npx envinfo --system --binaries --browsers --npmPackages="@grafana/*"` where you are experiencing the bug and paste the results here.
```
Comment (jackw) [inline .github/ISSUE_TEMPLATE/1-bug_report.md:24]: The command mentioned here will output the following:

```shell
  System:
    OS: macOS 12.6
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 76.14 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.17.1 - ~/.nvm/versions/node/v16.17.1/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.15.0 - ~/.nvm/versions/node/v16.17.1/bin/npm
  Browsers:
    Chrome: 106.0.5249.119
    Firefox: 104.0.2
    Safari: 16.0
  npmPackages:
    @grafana/data: latest => 9.2.0
    @grafana/toolkit: latest => 9.2.0
    @grafana/ui: latest => 9.2.0
```

File: .github/ISSUE_TEMPLATE/1-bug_report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Report a bug you found
```
Comment (academo) [inline .github/ISSUE_TEMPLATE/1-bug_report.md:3]: Given this is a mono-repo. We can add a field for project or package name. e.g.: create-plugin, sign-plugin and other upcoming tools.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,42 +1,149 @@
-# Contributing to Grafana / Create Plugin
+# Contributing
```
Comment (academo) [inline CONTRIBUTING.md:1]: We should include something about the CLA here.

**Comment by academo** [comment]
This is great! thanks for updating all these docs and adding even more. 

Only one question, do we also need to update the repository field in the several package.json? https://github.com/grafana/plugin-tools/blob/d3dda7395a875e02f4060cfc4decce8ff99c7078/package.json#L4 https://github.com/grafana/plugin-tools/blob/d3dda7395a875e02f4060cfc4decce8ff99c7078/packages/sign-plugin/package.json#L7 https://github.com/grafana/plugin-tools/blob/d3dda7395a875e02f4060cfc4decce8ff99c7078/packages/create-plugin/package.json#L7

File: .github/ISSUE_TEMPLATE/1-bug_report.md (hunk 2 above)
Comment (jackw) [reply .github/ISSUE_TEMPLATE/1-bug_report.md]: I've migrated to using github forms for this issue template.

**Review by academo — APPROVED** [review]
Thanks for putting this together

**Pull request merged**